imagepuller/internal/service/verify_test.go

```diff
@@ -29,6 +27,10 @@ func (s *StubStore) PutLayer(_, _ string, _ []string, _ string, _ bool, _ *stora
 }
 
 func TestGetAndVerifyImage_EvilRegistry(t *testing.T) {
+	registry := registry.New(t)
+	server := httptest.NewServer(registry)
+	t.Cleanup(server.Close)
```

Comment:
why move this out of the test case? that's global state between test cases!

Reply:
It's not mutable state, though, and I need the instantiated registry in order to know the digests for the test matrix. I think it's better than hard-coding digests, and also better than the `init` method currently used by `burgerdev/evil-registry`.

Reply:
Implemented, ptal.

Reply:
Thanks, sorry for the slow response.